client/src/actions/notification.js

```diff
@@ -0,0 +1,33 @@
+function guid() {
```

Comment:
We don't even need atomicity. We can use a monotonically incrementing int. I was debating that vs. guid, dont think it matters really. I'll change it.